### PR TITLE
chore: Script to generate root versions.txt

### PR DESCRIPTION
**Pull request by ddixit14**
This script generates versions.txt at the root. This is required for single-component setup. 
File generated by the script: [versions.txt](https://github.com/googleapis/google-cloud-java/files/9965592/versions.txt)

### CODE REVIEW

**Comment by ddixit14** [comment]
This is the file generated by the script: [versions.txt](https://github.com/googleapis/google-cloud-java/files/9965592/versions.txt)

**Review by meltsufin — APPROVED** [review]

File: generation/generate_root_versions_txt.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+set -e
+
+GENERATION_DIR=$(dirname -- "$0")
+
+echo "# Format:" >> versions.txt
```
Comment (suztomo) [inline generation/generate_root_versions_txt.sh:7]: Why is it appending? Is there a logic to empty the file first?

**Comment by suztomo** [comment]
Do you think we will run this script periodically?

If this is one-time change, I don't think you need to commit it to the repository.

**Review by suztomo — APPROVED** [review]

**Comment by suztomo** [comment]
jar-parent is there. Good.

**Comment by meltsufin** [comment]
> Do you think we will run this script periodically?

> If this is one-time change, I don't think you need to commit it to the repository. (no harm adding it; just remember to remove it few months later)

We might be adding more split repos to this repository. So, it might be a good idea to keep scripts like this one for the future.

**Comment by ddixit14** [comment]
> Do you think we will run this script periodically?
> 
> If this is one-time change, I don't think you need to commit it to the repository. (no harm adding it; just remember to remove it few months later)

We will need to run this everytime a new library/module is added to the monorepo.

File: generation/generate_root_versions_txt.sh (hunk 1 above)
Comment (ddixit14) [reply generation/generate_root_versions_txt.sh:9]: Oh yes, I forgot to remove the file if it exists before creating it. fixed now.

**Pull request merged**